packages/@aws-cdk/aws-rds/README.md

```diff
@@ -87,3 +120,13 @@ The `importedSecret` must be a JSON string with the following format:
   "port": "<optional: if not specified, default port will be used>"
 }
 ```
+
+### Metrics
+Database instances expose [metrics (cloudwatch.Metric)](https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-cloudwatch/README.md):
```

Comment:
Not a huge fan of linking this way (especially as it'll link to a possibly un-released version of the `README`, which might create confusion). I see why you'd want to do that however, and this might be a gap in our documentation story!

Reply:
I agree